src/context/soknadContextCommon.tsx

```diff
@@ -15,6 +15,8 @@ export const soknadContextInititalState = {
   type: undefined,
   currentStep: undefined,
   søknad: undefined,
+  requiredVedlegg: [],
+  søknadUrl: undefined,
```

Comment:
Denne ble lagt til tidligere for at vi skulle kunne få med lenke til søknad på kvitteringssiden. Må se om vi kan finne en bedre løsning, siden URLen vil bli borte hvis kvitteringssiden lastes på nytt.